src/com/carolinarollergirls/scoreboard/core/Team.java

```diff
@@ -51,6 +51,9 @@
     public int getScore();
 
     public ScoringTrip getCurrentTrip();
+    
+    //returns true if an advancement was pending
+    public boolean cancelTripAdvancement();
```

Comment:
This is an internal API for testing, no need to make it part of the public interface. The tests can already access it if they downcast to a TeamImpl.

Reply:
I'd say a case could be made for having this as part of the public API. But that's not a hill I'm even getting wounded on. Removed.